flask_apispec/wrapper.py

```diff
@@ -48,8 +54,9 @@ def marshal_result(self, unpacked, status_code):
         schemas = utils.merge_recursive(annotation.options)
         schema = schemas.get(status_code, schemas.get('default'))
         if schema and annotation.apply is not False:
-            schema = utils.resolve_schema(schema['schema'], request=flask.request)
-            output = schema.dump(unpacked[0]).data
+            schema = utils.resolve_instance(schema['schema'])
```

Comment:
This will need to be changed to `schema = utils.resolve_schema(schema['schema'], request=flask.request)`. `resolve_instance` was recently removed.